modules/mono/csharp_script.cpp

```diff
@@ -951,8 +951,14 @@ void CSharpLanguage::reload_assemblies(bool p_soft_reload) {
 #else
 				CRASH_COND(si != nullptr);
 #endif
-				// Re-create script instance
-				obj->set_script(script); // will create the script instance as well
+
+				// Re-create the script instance, bypassing the tool requirement
```

Comment:
Can you explain in the comment explaining why this was done? I can already see myself in two years confused about this code. A "see" link to this PR would be enough.